release-content/0.16/release-notes/17840_Parallel_Transform_Propagation.md

```diff
@@ -1,4 +1,31 @@
-<!-- Parallel Transform Propagation -->
-<!-- https://github.com/bevyengine/bevy/pull/17840 -->
+Transforms in Bevy (and other 3D software) come in two flavors:
 
-<!-- TODO -->
+1. Global transforms ([`GlobalTransform`]), which represent the absolute world-space position, scale and rotation of an object.
+2. Local transforms ([`Transform`]), which represent the position, scale and rotation of an object relative to its parent.
+
+In order to compute the global transform of each object (which is what rendering and physics care about!),
+we need to recursively combine the local transforms of all of our objects down the parent-child hierarchy.
+This process, known as transform propagation, has always been one of the most computationally intensive operations in most Bevy applications.
+Turns out, virtually everything in your world is going to have a transform!
```

Comment:
```suggestion
As it turns out, most entities in your game are going to have a transform :)
```

Could stick with a ! instead of :), not sure how to indicate tongue-in-cheek better.

Reply:
```suggestion
As it turns out, virtually everything in your world is going to have a transform!
```